applications/luci-app-squid/htdocs/luci-static/resources/view/squid.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+'require form';
+'require fs';
+'require rpc';
+'require uci';
+'require view';
+
+var getOptions = rpc.declare({
+	object: 'luci.squid',
+	method: 'getOptions',
```

Comment:
Can you rename this method to `getCompileTimeOptions` please, and have those changes reflect in the `luci.squid` file also?

Reply:
Absolutely.

Reply:
I'm really tempted to split this whole rpc method out and make it generic, so it can scrape any named executable. If you feel up to that bit, you're welcome, otherwise I can tackle it after merge.

Reply:
Feel free to do it after the merge, I think it's a good idea overall but a bit out-of-scope for the conversion part.

Reply:
Done.